backend/__init__.py

```diff
@@ -21,7 +30,8 @@
     ),
     Middleware(DatabaseMiddleware),
     Middleware(AuthenticationMiddleware, backend=JWTAuthenticationBackend()),
-    Middleware(ProtectedDocsMiddleware)
+    Middleware(ProtectedDocsMiddleware),
+    Middleware(SentryAsgiMiddleware),
```

Comment:
Should we put this higher up the middleware stack to catch middleware errors? Maybe after the auth middleware.